apps/frontend/src/app/5_pages/StakePage/components/PersonalStakingStatistics/hooks/useGetPersonalStakingStatistics.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useAccount } from '../../../../../../hooks/useAccount';
+import { useGetProtocolContract } from '../../../../../../hooks/useGetContract';
+import { asyncCall } from '../../../../../../store/rxjs/provider-cache';
+import { getRskChainId } from '../../../../../../utils/chain';
+import { useGetStakingBalanceOf } from '../../../hooks/useGetStakingBalanceOf';
+
+export const useGetPersonalStakingStatistics = () => {
+  const { account } = useAccount();
+  const stakingContract = useGetProtocolContract('staking', getRskChainId());
```

Comment:
```suggestion
  const stakingContract = useGetProtocolContract('staking');
```
`defaultChainId` is the default value for `chain` parameter so no need to provide that (unless you need to provide a custom signer as well).